### DISCUSSION TITLE
Allow sorting images by upload date and time

### DISCUSSION THREAD

**Discussion by tobiasge** [ente-io/ente#3846] [Enhancements]
When sharing an album with friends I get notifications when new photos are added, but is it not easy to find those new photos.
So I would like to sort the photos by upload date to see the newest on top.

**Comment by huntfx** [comment]
This would also be helpful if you accidentally uploaded the wrong photos. 

I just had to delete 10,000+ photos and reupload them, as I'd uploaded about 100 full quality photos rather than the compressed versions, so without a way to track them down and delete them, the only option really is to nuke the entire lot.
